### PR TITLE
Remove preview suffix from nightly READMEs

### PR DESCRIPTION
**Pull request by lbussell**
This was left out of https://github.com/dotnet/dotnet-docker/pull/4817

### CODE REVIEW

**Review by mthalman — APPROVED** [review]
We'll need this ported to main today as well.

**Comment by lbussell** [comment]
> We'll need this ported to main today as well.

Main READMEs don't have 8.0 featured tags yet. Should we make that change too? https://github.com/dotnet/dotnet-docker/blob/main/README.aspnet.md#featured-tags

File: eng/readme-templates/FeaturedTags.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ elif match(SHORT_REPO, "monitor"):* `7` (Standard Support)
 * `6` (Long-Term Support)
   * `docker pull {{FULL_REPO}}:6`^
 elif match(REPO, "monitor/base"):* `8` (Preview)
-  * `docker pull {{FULL_REPO}}:8-preview`^
+  * `docker pull {{FULL_REPO}}:8`^
 else:{{if VARIABLES["branch"] = "nightly":* `8.0` (Preview)
```
Comment (mthalman) [inline eng/readme-templates/FeaturedTags.md:17]: We should change these to be `RC` instead of `Preview`.

**Comment by mthalman** [comment]
> > We'll need this ported to main today as well.
> 
> Main READMEs don't have 8.0 featured tags yet. Should we make that change too? https://github.com/dotnet/dotnet-docker/blob/main/README.aspnet.md#featured-tags

Yes, I think we want them featured now in main, with the RC release.

**Review by mthalman — APPROVED** [review]

**Pull request merged**